tests/test_openclhpp.cpp

```diff
@@ -332,7 +332,7 @@ MAKE_REFCOUNT_STUBS(cl_command_queue, clRetainCommandQueue, clReleaseCommandQueu
 MAKE_REFCOUNT_STUBS(cl_device_id, clRetainDevice, clReleaseDevice, deviceRefcounts)
 MAKE_REFCOUNT_STUBS(cl_context, clRetainContext, clReleaseContext, contextRefcounts)
 MAKE_REFCOUNT_STUBS(cl_mem, clRetainMemObject, clReleaseMemObject, memRefcounts)
-#if defined(cl_khr_command_buffer)
+#if defined(cl_khr_command_buffer) && 0
```

Comment:
Yes those are unused for now, but I don't know if test using them may be needed soon. So I deactivated them rather than delete them altogether. It is not a very complex piece of code so I guess we could remove it for now as well.

Reply:
So that we don't wonder about the purpose of disabling it in the future, maybe it's worth adding a comment here?

Reply:
```suggestion
// Deactivated because unused for now.
#if defined(cl_khr_command_buffer) && 0
```
Would that work?

Reply:
It's good for me. Thanks for the change.